openpype/lib/vendor_bin_utils.py

```diff
@@ -120,6 +120,26 @@ def get_oiio_tools_path(tool="oiiotool"):
     return find_executable(os.path.join(oiio_dir, tool))
 
 
+def get_redshift_tool(tool_name):
+    """Path to redshift texture processor.
+
+    On Windows it adds .exe extension if missing from tool argument.
+
+    Args:
+        tool (string): Tool name.
+
+    Returns:
+        str: Full path to redshift texture processor executable.
+    """
+    redshift_tool_path = os.path.join(
+        os.environ["REDSHIFT_COREDATAPATH"],
+        "bin",
+        tool_name
+    )
+
+    return find_executable(redshift_tool_path)
```

Comment:
Not a huge issue - but this does feel off here since unlike the rest of the methods here this Redshift binary isn't actually vendorized? I do like that there's unison in the function itself however - nonetheless it's not perfect.

Reply:
Thank you @BigRoy, you bring up a good point.  I wonder where it would be good to place the function then @antirotor @iLLiCiTiT would love your take on where I should be placing this

Reply:
I would probably define separate ENV var for the tool that should be defined in the tool itself to use it.

Reply:
> I would probably define separate ENV var for the tool that should be defined in the tool itself to use it.

is it necessary? why can't we use the mandatory  `{REDSHIFT_COREDATAPATH}/bin/redshiftTextureProcessor.exe ` (and the appropriate lnx and mac variant)